<ui version="4.0" >
 <class>Qt_RealtimeIO_AppClass</class>
 <widget class="QMainWindow" name="Qt_RealtimeIO_AppClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>425</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>5</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font" >
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>Qt Realtime I/O</string>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item rowspan="2" row="0" column="1" >
     <widget class="QTextEdit" name="textEdit" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>3</hsizetype>
        <vsizetype>7</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>340</width>
        <height>0</height>
       </size>
      </property>
      <property name="font" >
       <font>
        <family>Courier New</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="readOnly" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" >
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string>Control Realtime I/O</string>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" colspan="2" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Clock Rate (msec)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QSpinBox" name="clockRate" >
         <property name="alignment" >
          <set>Qt::AlignRight</set>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="value" >
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2" >
        <widget class="QComboBox" name="processLevel" />
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Process Level</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Thread Level</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2" >
        <widget class="QComboBox" name="threadLevel" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Clock Mode</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2" >
        <widget class="QComboBox" name="clockMode" />
       </item>
       <item row="6" column="1" colspan="2" >
        <widget class="QPushButton" name="stopButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" >
        <widget class="QPushButton" name="startButton" >
         <property name="text" >
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Actual Rate</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2" >
        <widget class="QLineEdit" name="actualRate" >
         <property name="alignment" >
          <set>Qt::AlignRight</set>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QCheckBox" name="sendUDP" >
         <property name="text" >
          <string>Send UDP</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2" >
        <widget class="QLineEdit" name="sendIpAddr" >
         <property name="text" >
          <string>192.168.1.80</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>331</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
